tests/test_suite_test.go

```diff
@@ -14,13 +14,54 @@ limitations under the License.
 package tests
 
 import (
+	"os"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	"kubevirt.io/client-go/kubecli"
 )
 
 func TestKubevirtDpdkCheckup(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "KubevirtDpdkCheckup Suite")
 }
+
+const (
+	namespaceEnvVarName = "TEST_NAMESPACE"
+	imageEnvVarName     = "TEST_IMAGE"
+)
+
+const (
+	defaultNamespace = "kiagnose-demo"
+	defaultImageName = "quay.io/kiagnose/kubevirt-dpdk-checkup:latest"
+)
+
+var (
+	VirtClient    kubecli.KubevirtClient
```

Comment:
Is there a need for this variable to be exported?

Reply:
no actually